latex/project-report.tex

```diff
@@ -562,22 +558,23 @@ \section{Conclusion}
 SSH keys.
 
 Next, we did a survey of the state of the code then and now to see if
-the introduction of such code was due to incompetence on the debian
+the introduction of such code was due to carelessness on the debian
 developer's part, or an understandable error that no automated tools
 were in place to catch. We concluded that there was no easy way to
 tell if the code modification would adversely affect the security of
-the library, and that, furthermore, the debian developer was given the
-go-ahead.
+the library, and OpenSSL currently has solid tests in place to mitigate
+the risk.
 
-The current state of the code is much better however, having more
+The current state of the code is much better; however, having more
 thorough unit-testing coverage that can catch more accidental errors
-made by OpenSSL developers or package distributors. Furthermore, the
+made by OpenSSL developers or package distributors could further
```

Comment:
This changes the meaning of the sentence. The original sentence:

"The current state of the code is much better however, having more thorough unit-testing coverage that can catch more accidental errors made by OpenSSL developers or package distributors."

'however' here means 'despite the previous sentences', in all meaning 'despite what we found in the old code, the new code is better'. 
 
The new sentence:

"The current state of the code is much better; however, having more thorough unit-testing coverage that can catch more accidental errors made by OpenSSL developers or package distributors could further improve OpenSSL."

Partially condemns the current state of OpenSSL. 

I don't disagree with the new sentence but I prefer the old one because it is more positive.